tools/mo/openvino/tools/mo/moc_frontend/serialize.py

```diff
@@ -38,26 +38,5 @@ def moc_emit_ir(ngraph_function: Model, argv: argparse.Namespace):
         from openvino.tools.mo.back.offline_transformations import compress_model
         compress_model(ngraph_function)
 
-    orig_model_name = os.path.normpath(os.path.join(output_dir, argv.model_name))
-
-    from openvino.runtime import serialize # pylint: disable=import-error,no-name-in-module
-    from openvino.offline_transformations import generate_mapping_file # pylint: disable=import-error,no-name-in-module
-    serialize(ngraph_function, (orig_model_name + ".xml").encode('utf-8'), (orig_model_name + ".bin").encode('utf-8'))
-
     del argv.feManager
-
-    path_to_mapping = orig_model_name + ".mapping"
-    extract_names = argv.framework in ['tf', 'mxnet', 'kaldi']
-    generate_mapping_file(ngraph_function, path_to_mapping.encode('utf-8'), extract_names)
-
-    # add meta information to IR
-    append_ir_info(file=orig_model_name,
-                   meta_info=get_meta_info(argv),
-                   mean_data=None,
-                   input_names=None,
-                   legacy_path=False)
```

Comment:
Unfortunately we still need to call this from within MO to continue generate meta info until we resolve meta info representation in ov::Model.

Reply:
Done.

Reply:
Done.